components/form/input.tsx

```diff
@@ -54,6 +54,7 @@ export default function Input({
               placeholder={placeholder}
               type={type}
               {...field}
+              maxLength={100000}
```

Comment:
Looks like this should be `10000`?

Reply:
Yes.. you're right. It was meant to be 10,000.

Will change it to 500 though as it seems like a sensible upper limit on text inputs. 10,000 was just to save the DB.